### PR TITLE
Update logentries certs

### PR DESCRIPTION
**Pull request by k2nr**
Contacted with logentries. The suggested solution is to use OS CA bundle instead of `logentries.all.crt`

I confirmed OS CA bundle worked in a Amazon linux EC2 instance

### CODE REVIEW

**Comment by degicat** [comment]
@rramsden Please review this.

File: logentries/recipes/default.rb [hunk 1]
```diff
@@ -7,20 +7,13 @@
...
-end
-
 logentries = node[:deploy].map{ |e| e[1][:logentries] }.compact.first
 
 template '/etc/rsyslog.d/logentries.conf' do
   source 'logentries.conf.erb'
   owner 'root'
   group 'root'
-  mode '0755'
+  mode '0644'
```
Comment (k2nr) [inline logentries/recipes/default.rb]: Not related to the issue but I found this permission is not correct. I don't think this conf file needs x permission

File: logentries/templates/default/logentries.conf.erb [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 $ActionSendStreamDriver gtls
 $ActionSendStreamDriverMode 1
 $ActionSendStreamDriverAuthMode x509/name
 $ActionSendStreamDriverPermittedPeer *.logentries.com
 
-$template LogentriesFormat,"<%= @token %> %HOSTNAME% %syslogtag%%msg%\n"
-*.* @@api.logentries.com:20000;LogentriesFormat
+$template LogentriesTemplate,"<%= @token %> %HOSTNAME% %syslogtag%%msg%\n"
+*.* @@data.logentries.com:443;LogentriesTemplate
```
Comment (k2nr) [inline logentries/templates/default/logentries.conf.erb]: Logentries support said api is a deprecated endpoint and in https://docs.logentries.com/docs/rsyslog#section-secure-logging the example config uses data

**Comment by degicat** [comment]
@rramsden Please review this.

**Comment by degicat** [comment]
@camelmasa Please review this.

**Comment by camelmasa** [comment]
@k2nr LGTM 👍

File: logentries/files/default/logentries.all.crt [hunk 3]
```diff
@@ -1,55 +0,0 @@
------BEGIN CERTIFICATE-----
```
Comment (k2nr) [inline logentries/files/default/logentries.all.crt]: this is my first review

**Comment by k2nr** [comment]
Thanks I'll test on sandbox

**Pull request merged**